### PR TITLE
Add Kunal0007 to the list

### PR DESCRIPTION
**Pull request by Kunal0007**
Fixes #

## Proposed Changes

  -
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/179"></a>

### CODE REVIEW

**Review by voidz7 — COMMENTED** [review]
Looks good ⚡  , btw you can use better commit messages like #89

**Review by voidz7 — APPROVED** [review]

**Review by qwe123coder — APPROVED** [review]
nice work :rocket: 
LGTM :rocket:

**Review by Nikhil-1503 — APPROVED** [review]

**Comment by Kunal0007** [comment]
@schmelto when it will get merged?

File: public/data/Kunal0007.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "avatar": "https://github.com/Kunal0007.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Kunal0007",
+        "icon": "github"
+      }
+    ]
+  }
+
```
Comment (eddiejaoude) [inline public/data/Kunal0007.json:13]: needs a blank line ending at the end of the file (see the red mark by GitHub)

Comment (Kunal0007) [reply public/data/Kunal0007.json]: Added blank line at the end of the file

File: public/data/Kunal0007.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "bio": "I am Kunal Patil currently pursuing a BTech degree in CS at PCCOE - Pimpri Chinchwad College Of Engineering, Pune. A Frontend Developer | Open-Source Enthusiast",
+    "avatar": "https://github.com/Kunal0007.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Kunal0007",
+        "icon": "github"
+      }
+    ]
+  }
```
Comment (FayasNoushad) [inline public/data/Kunal0007.json:12]: Two white spaces before `}`. Its not needed.

**Review by FayasNoushad — APPROVED** [review]

File: public/data/Kunal0007.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "avatar": "https://github.com/Kunal0007.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Kunal0007",
+        "icon": "github"
+      }
+    ]
+}
+
```
Comment (Panquesito7) [inline public/data/Kunal0007.json:13]:
```suggestion
```

File: public/data/Kunal0007.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Kunal Patil",
+    "bio": "I am Kunal Patil currently pursuing a BTech degree in CS at PCCOE - Pimpri Chinchwad College Of Engineering, Pune. A Frontend Developer | Open-Source Enthusiast",
+    "avatar": "https://github.com/Kunal0007.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Kunal0007",
```
Comment (Panquesito7) [inline public/data/Kunal0007.json:8]:
```suggestion
        "url": "https://github.com/Kunal0007",
```

File: public/data/Kunal0007.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Kunal Patil",
+    "bio": "I am Kunal Patil currently pursuing a BTech degree in CS at PCCOE - Pimpri Chinchwad College Of Engineering, Pune. A Frontend Developer | Open-Source Enthusiast",
```
Comment (Panquesito7) [inline public/data/Kunal0007.json:3]: Maybe provide a shorter description of yourself? It is kind of long.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/Kunal0007.json (hunk 3 above)
Comment (Kunal0007) [reply public/data/Kunal0007.json:13]: @eddiejaoude told needs a blank line ending at the end of the file

**Review by voidz7 — APPROVED** [review]

File: public/data/Kunal0007.json (hunk 3 above)
Comment (Panquesito7) [reply public/data/Kunal0007.json:13]: The Git diff doesn't display the last line. So you have 2 extra lines here that should be removed. 🙂

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**